index.js

```diff
@@ -7,8 +7,11 @@ const PORT = process.env.PORT || 8000;
 app.use(bodyParser.json());
 
 const directions = require('./src/routes/directions');
+const alerts = require('./src/routes/alerts');
 
 app.use(directions);
+app.use(alerts);
+
 
 app.get('/', (req, res) => {
```

Comment:
@Sanderson239 Also I think probably you could remove this boilerplate code, thanks!